magento/magento2
# Improve dashboard charts - migrate to chart.js

### Description (*)

This PR aims to migrate dashboard charts to chart.js library instead of static images.
Fixes magento/community-features#186
This is preview of new charts:
![image](https://user-images.githubusercontent.com/9566840/74825715-ab7b8a00-530a-11ea-9cfc-4accb18afeea.png)

### Related Pull Requests


### Manual testing scenarios (*)

1. Enable Dashboard Charts
2. Place some orders with different dates and amounts, process them to `Complete` status
3. Go to MBO Dashboard
4. Dynamic charts should be visible 

### Questions or comments


### Contribution checklist (*)
